### ISSUE TITLE
Error thrown when react-spreadsheet is imported

### ISSUE THREAD

**Issue by ankitgoyal100** [iddan/react-spreadsheet#95]
```
ERROR in ./node_modules/react-spreadsheet/dist/index.mjs 21:0-60
Module not found: Error: Can't resolve 'hot-formula-parser/lib/helper/cell' in '/Users/project/node_modules/react-spreadsheet/dist'
Did you mean 'cell.js'?
BREAKING CHANGE: The request 'hot-formula-parser/lib/helper/cell' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./frontend/src/components/Upload.tsx 16:0-44 84:40-51 86:40-51
 @ ./frontend/src/App.tsx 19:0-41 49:133-139
 @ ./frontend/src/index.tsx 3:0-24 9:28-31
```

**Comment by iddan** [comment]
Thank you for reporting, which tools are you using to build your project?

**Comment by ankitgoyal100** [comment]
Here is my `package.json`:
```
{
  "name": "dromo-frontend",
  "version": "1.0.0",
  "description": "",
  "scripts": {
    "start": "webpack --mode development --config webpack.config.js",
    "watch": "webpack --mode development --config webpack.config.js --watch",
    "build": "node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js --mode=production",
    "build:stats": "webpack --mode production --json > stats.json",
    "build:analyze": "webpack-bundle-analyzer stats.json"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@types/react": "^16.9.27",
    "@types/react-dom": "^16.9.7",
    "awesome-typescript-loader": "^5.2.1",
    "clean-webpack-plugin": "^3.0.0",
    "compression-webpack-plugin": "^6.1.1",
    "css-loader": "^5.0.1",
    "mini-css-extract-plugin": "^1.3.1",
    "source-map-loader": "^1.1.2",
    "typescript": "^4.1.2",
    "webpack": "^5.6.0",
    "webpack-bundle-tracker": "^1.0.0-alpha.1",
    "webpack-cli": "^4.2.0",
    "webpack-dev-server": "^3.11.0"
  },
  "dependencies": {
    "axios": "^0.21.0",
    "bootstrap": "^4.5.3",
    "history": "^5.0.0",
    "node-sass": "^5.0.0",
    "papaparse": "^5.3.0",
    "react": "^17.0.1",
    "react-bootstrap": "^1.4.0",
    "react-dom": "^17.0.1",
    "react-redux": "^7.2.2",
    "react-router-dom": "^5.2.0",
    "react-spreadsheet": "^0.5.0",
    "redux": "^4.0.5",
    "redux-thunk": "^2.3.0"
  }
}
```

**Comment by iddan** [comment]
This is solved with `react-spreadsheet@0.8.0`
